components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3375,6 +3375,12 @@ subroutine cloud_sedimentation(kts,kte,ktop,kbot,kdir,   &
 
             call generalized_sedimentation(kts, kte, kdir, k_qxtop, k_qxbot, kbot, Co_max, dt_left, prt_accum, inv_dzq, inv_rho, rho, num_arrays, vs, fluxes, qnr)
 
+            !Update _incld values with end-of-step cell-ave values
+            !Note that lcldm is set in interface to have min of mincld=1e-4
+            !so dividing by it is fine.
+            qc_incld(:) = qc(:)/lcldm(:)
+            nc_incld(:) = nc(:)/lcldm(:)
+
```

Comment:
As the cloud drop falls out of what was originally the cloud into areas where lcldm =0, I think lcldm needs to be updated so that qc_incld and nc_incld don't blow up. The same goes for qr, qi, qirim below.